app-tracking-protection/vpn-internal/src/main/res/layout/activity_vpn_internal_settings.xml

```diff
@@ -90,11 +90,11 @@
             />
 
             <androidx.appcompat.widget.SwitchCompat
-                    android:id="@+id/vpnUnderlyingNetworksToggle"
+                    android:id="@+id/vpn_intercept_dns_traffic_toggle"
```

Comment:
Just pointing out, this class has a mix of majority camel case and snake case. Would've been good to have only one.